.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "sources/cumulus-library-core"]
 	path = sources/cumulus-library-core
 	url = https://github.com/smart-on-fhir/cumulus-library-core
```

Comment:
while we're here - can we change this to the new library url?

Reply:
Yeah makes sense and done!

(while doing this, I noticed that the library's pyproject.toml still has a link to -core, FYI)